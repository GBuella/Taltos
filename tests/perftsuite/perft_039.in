setboard K7/8/2n5/1n6/8/8/8/k6N w - - 0 1
echo K7/8/2n5/1n6/8/8/8/k6N w - - 0 1
qperft 1
qperft 2
qperft 3
qperft 4
qperft 5
qperft 6
