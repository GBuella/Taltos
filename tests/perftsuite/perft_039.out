K7/8/2n5/1n6/8/8/8/k6N w - - 0 1
3
51
345
5301
38348
588695
